src/status_im2/contexts/shell/jump_to/components/switcher_cards/view.cljs

```diff
@@ -34,67 +35,69 @@
    {:keys                             [content-type data new-notifications? color-50
                                        community-info community-channel]
     {:keys [text parsed-text source]} :data}]
-  [rn/view {:style (style/content-container new-notifications?)}
-   (case type
-     shell.constants/community-card
-     (case (:type community-info)
-       :pending             [quo/status-tag
-                             {:status {:type :pending}
-                              :label  (i18n/label :t/pending)
-                              :size   :small}]
-       :kicked              [quo/status-tag
-                             {:status {:type :negative}
-                              :size   :small
-                              :label  (i18n/label :t/kicked)}]
-       (:count :permission) [:<>] ;; Add components for these cases
-       nil)
-
-     shell.constants/community-channel-card
-     [channel-card (assoc community-channel :customization-color color-50)]
-
-     (case content-type
-       constants/content-type-text
-       [quo/text
-        {:size            :paragraph-2
-         :weight          :regular
-         :number-of-lines 1
-         :ellipsize-mode  :tail
-         :style           style/last-message-text}
-        (if parsed-text
-          (resolver/resolve-message parsed-text)
-          text)]
-
-       constants/content-type-image
-       [quo/preview-list
-        {:type               :photo
-         :more-than-99-label (i18n/label :counter-99-plus)
-         :size               24}
-        data]
-
-       constants/content-type-sticker
-       [fast-image/fast-image
-        {:source source
-         :style  style/sticker}]
-
-       constants/content-type-gif
-       [fast-image/fast-image
-        {:source source
-         :style  style/gif}]
-
-       constants/content-type-audio
-       [quo/audio-tag data {:override-theme :dark}]
-
-       constants/content-type-community
-       [quo/community-tag
-        (:avatar data)
-        (:community-name data)
-        {:override-theme :dark}]
-
-       (constants/content-type-link) ;; Components not available
-       ;; Code snippet content type is not supported yet
-       [:<>]
-
-       nil))])
+  [quo.theme/provider {:theme :dark}
```

Comment:
Instead of `:override-theme :dark`, I'm using our theme provider and wrapping the components

Reply:
Fixed!